tests/integration/targets/nxos_ospfv2/tests/common/_populate_config.yaml

```diff
@@ -0,0 +1,26 @@
+- name: Setup
+  cisco.nxos.nxos_config:
+    lines:
```

Comment:
IMO it's better to use overridden state for setting configuration instead of line

Reply:
We shouldn't be using the same module for setup as the one we are testing.